tools/clang/test/HLSLFileCheck/hlsl/compile_options/precise_output.hlsl

```diff
@@ -0,0 +1,17 @@
+// RUN: %dxc -E main -T vs_6_0 -precise-output d1 -precise-output SV_Position %s | FileCheck %s
+
+// CHECK-NOT:fast
```

Comment:
If this fails to compile, the test succeeds.  Do we want a `CHECK: define void @main` first?